### ISSUE TITLE
Request for introduction of MixSIAR model theory including model fuctions

### ISSUE THREAD

**Issue by mymyabc5186** [brianstock/MixSIAR#34]
I did not find the introduction of model theory including model functions in the MixSIAR GUI User Manual. Could you send me a paper or a brief introduction? 

Thanks again!

**Comment by AndrewLJackson** [comment]
you are joking me right? All the papers with all the maths is listed here.. on the front page of this repo! https://github.com/brianstock/MixSIAR

**Comment by brianstock** [comment]
I agree that the User Manual is not as clear as it could be.  I'm planning on giving it an update, and when I do I'll more explicitly spell out (in math) how the different options create different mixing models.

For now:
The MixSIAR code can fit the same models as MixSIR and SIAR, but adds some additional features, particularly the GUI and the use of covariates, as:
  - random effects, based on the Semmens et al (2009) paper on British Columbia wolves
  - fixed effects, instead of running multiple separate SIAR or MixSIR models (although slightly different because only one residual error term is estimated per isotope instead of one per group as it would be if you run separate models in MixSIR/SIAR/MixSIAR)
  - continuous, as in Francis et al (2011) on lakes (just a simple linear regression in ILR space)

Other modifications to the plain vanilla MixSIR/SIAR models:
  - allow choice of MixSIR vs. SIAR error (SIAR adds a residual error term, MixSIR does not)
  - fit source data hierarchically within the model (Ward 2010 Env Sci & Tech, which was in the Environmetrics paper as well but differently).
  - use ILR transform

The implementation of covariates in MixSIAR is a linear model in ILR space, as in case study 2 of the 2013 Parnell et al. Environmetrics paper.

**Comment by brianstock** [comment]
Additionally, if you are ever unsure exactly what equations are being used, you can look at the "mixsiar_model.txt" file that is created when you run a model.  That is the model passed to JAGS.
